pysoa/common/transport/base.py

```diff
@@ -9,15 +9,16 @@
 All Transport methods either accept or return a metadata argument. This should be a
 dict that includes any information that is necessary for processing the message, but
 is not business logic. For example, if your implementation has multiple serializer
-types, the metadata may include a mimetype to tell the endpoint receiving the message
+types, the metadata may include a mime type to tell the endpoint receiving the message
 which type of serializer to use.
 """
 
 
 class ClientTransport(object):
 
-    def __init__(self, service_name):
+    def __init__(self, service_name, metrics):
```

Comment:
I would make `metrics` a keyword arg that default to `NoOpMetricsRecorder`, just for consistency. In `ServerTransport` as well.